fawkes/cli/cli.py

```diff
@@ -54,18 +56,34 @@ def define_arguments(parser):
         type=str,
         default=constants.FAWKES_CONFIG_FILE,
     )
+    # Specify query index for elasticsearch query
+    parser.add_argument(
+        "-q", "--query",
+        help="The query index for elasticSearch query",
+        type=str,
+        default="fawkes-user-reviews",
```

Comment:
Do we default to this index anywhere else ? I don't think so. Can you just put it as `None`, just let it fail if nothing is provided.

Reply:
Or we can try to make it a required argument when the command is `query.elasticsearch` which might make it complicated

Reply:
Either is fine by me.

Reply:
putting it as 'None' will throw an elasticSearch error
putting it as '' will retrieve all the indexes present
@MadaraUchiha-314

Reply:
Sure. Go ahead and make it ""